tests/searching/task.yaml

```diff
@@ -0,0 +1,38 @@
+summary: Check snap search
+execute: |
+  echo "List all snaps"
+  expected="(?s)Name +Version +Developer +Notes +Summary *\n\
+  .*?\n\
+  canonical-pc +.*? *\n\
+  .*?\
+  canonical-pc-linux +.*? *\n\
+  .*?\
+  go-example-webserver +.*? *\n\
+  .*?\
+  hello-world +.*? *\n\
+  .*?\
+  ubuntu-clock-app +.*? *\n\
+  .*?\
+  ubuntu-core +.*? *\n\
```

Comment:
Probably okay to keep ubuntu-clock-app there as well for the time being, at least to the ordering and the fact we're showing more than a single snap. But let's please try to constraint here and in other tests the number of snaps we assume exists on the store.